Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:20.14.0-alpine AS base
+FROM node:20-alpine AS base
```

Comment:
Would renovate overwrite this with a specific version later on?

Reply:
If the bot overwrites it afterwards, it is ok.

Reply:
I hope they’ve fixed the issue where this tag would mean "whatever matching version was cached on this host in the last months"

Reply:
Not sure about that either. 😅